### ISSUE TITLE
Make partner specific setting to check for duplication or not

### ISSUE THREAD

**Issue by chadgates** [abhishek-ram/django-pyas2#33]
Sometimes, a message transfer mail be successful but not processed as such by the partner. May connection being terminated before synchronous MDN sending has finished or that an MDN is not properly handled or ... 

In other AS2 solutions, I found the option to disabled duplicate check per partner. Not exactly sure what that does, but I would suggest a partner specific setting: 

- Option to send "positive" MDN for duplicate message, but handle internally like when duplicate message is received. 

Any thoughts on this?

**Comment by abhishek-ram** [comment]
Yes I think we should be allow sending positive MDN for duplicate messages. We can just make it a global setting to keep it simple

**Comment by abhishek-ram** [comment]
I just realized that right now we have duplicate check on just the message id and partner id, this means that if a partner sends a message and it failed and later they sent another message that would have succeeded it would still fail because of Duplicate check. I think this is a bug that we would need to fix.

**Comment by chadgates** [comment]
#37
